repo/packages/P/prometheus/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "prometheus",
+  "version": "0.15.1-0.1",
+  "scm": "https://github.com/prometheus/prometheus",
+  "maintainer": "support@mesosphere.io",
+  "description": "The Prometheus monitoring system and time series database.",
+  "framework": true,
+  "images": {
+    "icon-small": "https://downloads.mesosphere.io/prometheus/artifacts/images/prometheus-48x48.png",
+    "icon-medium": "https://downloads.mesosphere.io/prometheus/artifacts/images/prometheus-96x96.png",
+    "icon-large": "https://downloads.mesosphere.io/prometheus/artifacts/images/prometheus-256x256.png"
+  },
+  "tags": ["mesosphere", "monitoring", "time-series"],
```

Comment:
tags should not contain the words mesosphere, mesos, framework etc.  these are used for searches to help users find what they are looking for... those words are true for all things. (there are some bad examples currently in the universe which are slated to be fixed.